.github/PULL_REQUEST_TEMPLATE/ska3-flight.md

```diff
@@ -16,9 +16,11 @@ The latest release candidates will be installed in `/proj/sot/ska3/test` on HEAD
 conda create -n ska3-flight-{version}rc# --override-channels \
   -c https://icxc.cfa.harvard.edu/aspect/ska3-conda/flight \
   -c https://icxc.cfa.harvard.edu/aspect/ska3-conda/test \
-  ska3-flight=={version}rc#
+  ska3-flight=={version}rc# ska3-perl=={version}rc#
```

Comment:
For this, I suppose there's a legit process question.  We're making a new ska3-perl package for every flight release - should we just install it and save the question about "is there a perl package update?".  I'm suggesting here that we might as well.

Reply:
We rarely release ska3-perl, so most of the time this line will not work.

Reply:
Ah.  Sorry, I thought we were just always building this - so this should be fixed.  Will do.